integration_tests/test-environment/__tests__/env.test.js

```diff
@@ -5,7 +5,7 @@
  * LICENSE file in the root directory of this source tree. An additional grant
  * of patent rights can be found in the PATENTS file in the same directory.
  *
- * @jest-environment jsdom
+ * @jest-environment ./test-environment
```

Comment:
In a completely new file please. This test is for specifying `jsdom` in docblock whilst package.json specifies `node`